package.json

```diff
@@ -8,11 +8,18 @@
   },
   "author": "",
   "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/siarohin/learn-nodejs/issues"
+  "scripts": {
+    "babel-node": "babel-node --presets=@babel/preset-env",
+    "task1": "nodemon --exec npm run babel-node -- ./src/task1/index.js",
+    "task2": "nodemon --exec npm run babel-node -- ./src/task2/index.js"
```

Comment:
better to use single point of entry, use modular system of node js and import both tasks to main.js for instance.